Mathlib/Algebra/Order/Ring/Cast.lean

```diff
@@ -37,6 +37,8 @@ lemma cast_mono : Monotone (Int.cast : ℤ → R) := by
   rw [← sub_nonneg, ← cast_sub, ← hk, cast_natCast]
   exact k.cast_nonneg'
 
+@[gcongr] protected lemma GCongr.intCast_mono {m n : ℤ} (hmn : m ≤ n) : (m : R) ≤ n := cast_mono hmn
```

Comment:
GCongr.Int.cast_mono looks a bit more consistent with the others here

Reply:
I think this is dangerous because `open Int GCongr` would result in `cast_mono` being ambiguous between `Int.cast_mono` and `Int.GCongr.Int.cast_mono`